test/e2e/util/util.go

```diff
@@ -1194,6 +1199,7 @@ func EnsurePodsWithEmptyDirPVsHaveSafeToEvictAnnotations(t *testing.T, ctx conte
 			"redhat-marketplace-catalog":             "app",
 			"openstack-cinder-csi-driver-controller": "app",
 			"manila-csi-driver-controller":           "app",
```

Comment:
We could remove this line, no?

Reply:
you're very probably right, I was confused by:

https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_hypershift/5202/pull-ci-openshift-hypershift-main-e2e-openstack/1863871352625172480/artifacts/e2e-openstack/hypershift-openstack-e2e-execute/artifacts/TestCreateCluster/namespaces/e2e-clusters-x9kpt-example-mznq8/apps/deployments/openstack-manila-csi-controllerplugin.yaml

```
        podAntiAffinity:
          preferredDuringSchedulingIgnoredDuringExecution:
          - podAffinityTerm:
              labelSelector:
                matchLabels:
                  app: manila-csi-driver-controller
```

I'll remove it as a follow-up. We don't need to block the PR for that.